Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 134,
+        "Minor": 135,
```

Comment:
This is sprint based. So you can name it 136

Reply:
Done